package.json

```diff
@@ -29,6 +30,7 @@
     "fetch-mock": "^7.3.9",
     "file-loader": "^3.0.1",
     "husky": "^2.3.0",
+    "identity-obj-proxy": "^3.0.0",
```

Comment:
Adding this is OK, but I’m not in love with it. I feel like this is one of those modules that is too small to be worth the overhead and security risk of a dependency (see also: left-pad). You could replicate the whole functionality like:

```js
// File: src/__mocks__/identity-object.js
/**
 * An object where any key's value is the name of the key.
 * @example
 * identityObject.a === 'a';
 * identityObject.b === 'b';
 */
export default new Proxy({}, {
  get (target, key) {
    // Support Babel/Webpack CommonJS compilation, which uses __esModule.
    if (key === '__esModule') {
      return false;
    }
    return key;
  }
});
```

```js
// in package.json:
"jest": {
  "moduleNameMapper": {
    "\\.css$": "<rootDir>/src/__mocks__/identity-object.js"
  }
}
```

Reply:
I agree with this assessment. In cases like this, dependency management is more work than maintaining a small bit of custom code.

Reply:
That's fair, I will remove it.